rules/recording.go

```diff
@@ -112,6 +123,16 @@ func (rule *RecordingRule) SetEvaluationDuration(dur time.Duration) {
 	rule.evaluationDuration = dur
 }
 
+// Health returns the current health of the recording rule
+func (rule *RecordingRule) Health() RuleHealth {
+	return rule.health
```

Comment:
I think this and `LastError()` also need to be protected against concurrent access via the mutex, like in `GetEvaluationDuration()`?

While the web UI gets a copy of the list of rules, it's a copy of pointers.